Fun.Build/ConditionsBuilder.fs

```diff
@@ -32,6 +32,17 @@ module Internal =
                 false
             | Mode.Execution -> isTrue
 
+        member ctx.WhenStage(stage: StageContext) =
+            match ctx.GetMode() with
+            | Mode.Execution ->
+                let result, exns = stage.Run(StageIndex.WhenStage, System.Threading.CancellationToken.None)
+                result
+            | Mode.Verification ->
+                AnsiConsole.MarkupLineInterpolated($"[yellow]? Check results of WHEN STAGE {stage.Name} above.[/]")
```

Comment:
Can we change it to: [yellow]? [/]{ctx.BuildIndent().Substring(2)}when stage [green]{stage.Name}[/] finished successfully

Reply:
In this case it may not have finished successfully... if I understood correctly, this text is emitted if pipeline verification failed? Other verifications are evaluated based on static data so they can be rechecked when running in Verification mode, but for these stages if rerun the when' stage it will re-run the whole stage which may have unintended side effects.

Ideally it would be nice to capture the result somewhere so it can be used here but I wasn't sure how to do that...

Reply:
I know the when stage will not actually run when do verification. So the **?** is reserved. I should not use green color. Let's change it to: [yellow]? [/]{ctx.BuildIndent().Substring(2)}check results of stage [yellow]{stage.Name}[/]

Reply:
Yes, it was the green colour confusing me - yes, happy to go with this string.